1.1/learn/how-to-document-ballerina-code.md

```diff
@@ -64,6 +64,19 @@ The supported structure of documentation syntax is as follows:
 #            <return_parameter_description_line_2>
 ```
 
+>**Tip:** As a best practice, do not add a full stop at the end of the first sentence of the parameter and return type descriptions. Instead, add a fullstop only at the end of a function description. For Example,
+
+```
+# Description for the function.
+#
+# + i - One sentence only
+# + s - Sentence one. Sentence two.
+# + return - Return description
+public function foo(int i, string s) returns boolean {
+return true;
```

Comment:
The code is not properly formatted. Check all the places.

Reply:
@praneesha please load these sample code snippets to vscode, do a code format, and use it after that.

Reply:
Done.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/1269.